README.md

```diff
@@ -9,7 +9,7 @@
 |    | Sample Name        | Description                                                                                                                | C#    | TypeScript   
 |:--:|:-------------------|:---------------------------------------------------------------------------------------------------------------------------|:--------|:-------------|
 |1|Hello World            | Microsoft Teams hello world sample app.                                           |[View][app-hello-world#cs]     |[View][app-hello-world#ts]
-|2|App Complete Sample    | Sample covering multiple scenarios.                                               |[View][app-complete-sample#cs]     |
+|2|App Complete Sample    | Sample covering multiple scenarios.                                               |[View][app-complete-sample#cs]     |[View][app-complete-sample#js]
```

Comment:
Let's change this to "Sample covering multiple scenarios - dialogs, ME, and facebook auth"